python/taichi/main.py

```diff
@@ -815,6 +823,34 @@ def test(self, arguments: list = sys.argv[2:]):
                             dest='rerun',
                             type=str,
                             help='Rerun failed tests for given times')
+        parser.add_argument('-k',
+                            '--keys',
+                            required=False,
+                            default=None,
+                            dest='keys',
+                            type=str,
+                            help='Only run tests that matched the keys')
+        parser.add_argument('-m',
+                            '--marks',
+                            required=False,
+                            default=None,
+                            dest='marks',
+                            type=str,
+                            help='Only run tests with specific marks')
```

Comment:
What're marks?

Reply:
e.g.:
```
@pytest.mark.xumingkuan
def test_hello():
  ...
```
```bash
ti test -m xumingkuan
```
I'll add a doc later clarify the test infrastructure.